bucket/ghostwriter.json

```
@@ -1,17 +1,20 @@
 {
-    "homepage": "https://wereturtle.github.io/ghostwriter/",
+    "homepage": "https://github.com/michelolvera/vs-ghostwriter",
```

I really do not see a reson to change homepage to forked repository when it is only for binaires.
```suggestion
    "homepage": "https://wereturtle.github.io/ghostwriter/",
```